content/bn/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: কুবারনেটিসে অবজেক্ট
+content_type: concept
+weight: 10
+description: >
+   কুবারনেটিস অবজেক্ট হল কুবারনেটিস সিস্টেমে স্থায়ী সত্তা।
+   কুবারনেটিস আপনার ক্লাস্টারের অবস্থার প্রতিনিধিত্ব করতে এই সত্তাগুলি ব্যবহার করে।
+   কুবারনেটিস অবজেক্ট মডেল এবং এই বস্তুর সাথে কিভাবে কাজ করতে হয় সে সম্পর্কে জানুন।
+simple_list: true
+card:
+  name: concepts
+  weight: 40
+---
+
+<!-- overview -->
+
+এই পৃষ্ঠাটি ব্যাখ্যা করে কুবারনেটিস API-তে কুবারনেটিস অবজেক্টগুলি কীভাবে প্রতিনিধিত্ব করা হয় এবং 
+আপনি কিভাবে তা `.yaml` ফরম্যাটে প্রকাশ করতে পারেন।
+
+<!-- body -->
+
+## কুবারনেটিস অবজেক্ট বোঝা {#kubernetes-objects}
+
+*কুবারনেটিস অবজেক্ট* হল কুবারনেটিস সিস্টেমের সত্তা সংরক্ষিত এন্টিটিগুলি। কুবারনেটিস এই 
+এন্টিটিগুলি ব্যবহার করে আপনার ক্লাস্টারের অবস্থা প্রকাশ করতে। বিশেষভাবে, তারা বর্ণনা করতে পারে:
+
+- কোন কন্টেনার অ্যাপ্লিকেশন কি রান করছে (এবং কোন নোডগুলিতে)
+- ঐ অ্যাপ্লিকেশনগুলির জন্য উপলব্ধ সম্পদ
+- ঐ অ্যাপ্লিকেশনগুলির কিভাবে ব্যবহার করতে হবে, উদাহরণস্বরূপ পুনরারম্ভন নীতি, আপগ্রেড, এবং ত্রুটি সহ্যতা
+
+একটি কুবারনেটিস অবজেক্ট হল একটি "উদ্দেশ্যের রেকর্ড" - একবার আপনি অবজেক্ট তৈরি করে দিলে, 
+কুবারনেটিস সিস্টেম সরাসরি এই অবজেক্টটি থাকার নিশ্চয়তার জন্য কাজ করবে। অবজেক্ট তৈরি করে 
+আপনি সাধারণত কুবারনেটিস সিস্টেমকে বলে দিচ্ছেন যে আপনার ক্লাস্টারের ওয়ার্কলোড কি হবে; এটা 
+হল আপনার ক্লাস্টারের *কাঙ্ক্ষিত অবস্থা*।
+
+কুবারনেটিস অবজেক্টগুলির সাথে কাজ করতে - তা তৈরি, পরিবর্তন করতে বা মুছতে - আপনার 
+[কুবারনেটিস API](/docs/concepts/overview/kubernetes-api/) ব্যবহার করতে হবে। উদাহরণস্বরূপ, 
+যখন আপনি `kubectl` কমান্ড-লাইন ইন্টারফেস ব্যবহার করেন, তখন CLI আপনার জন্য প্রয়োজনীয় 
+কুবারনেটিস API কল করে। আপনি একটি [Client Libraries](/docs/reference/using-api/
+client-libraries/) ব্যবহার করে নিজের প্রোগ্রামে কুবারনেটিস API সরাসরি ব্যবহার করতে পারেন।
+
+### অবজেক্ট স্পেক এবং স্ট্যাটাস
+
+প্রায় সব কুবারনেটিস অবজেক্টের একটি `spec` এবং একটি `status` নেস্টেড অবজেক্ট ফিল্ড রয়েছে 
+যা অবজেক্টের কনফিগারেশন নিয়ন্ত্রণ করে: অবজেক্টের _`spec`_ এবং _`status`_। 
+যে অবজেক্টগুলির `spec` থাকে, আপনার অবজেক্ট তৈরি করতে এটা নির্ধারণ করতে হবে 
+যখন অবজেক্ট তৈরি করছেন, 
+যে কোন রিসোর্সের বৈশিষ্ট্য বর্ণনা প্রদান করে: এর _কাঙ্ক্ষিত অবস্থা_।
```

Comment:
যে অবজেক্টগুলির `spec` থাকে, আপনার অবজেক্ট তৈরি করতে এটা নির্ধারণ করতে হবে 
যখন অবজেক্ট তৈরি করবেন, 
 রিসোর্সের বৈশিষ্ট্য বর্ণনা প্রদান করবেন : এর _কাঙ্ক্ষিত অবস্থা_।

Reply:
` `  use this